frontend/src/metabase/visualizations/components/ChartClickActions.jsx

```diff
@@ -183,92 +198,105 @@ class ChartClickActions extends Component {
+                        SECTIONS[key].icon === "sum" ||
+                        SECTIONS[key].icon === "breakout" ||
+                        (SECTIONS[key].icon === "funnel_outline" &&
+                          !hasOnlyOneSection),
                     },
-                    {
-                      "align-center justify-center":
-                        SECTIONS[key].icon === "gear",
-                    },
-                    { "flex-column my1": SECTIONS[key].icon === "summarize" },
                   )}
                 >
-                  {actions.map((action, index) => (
-                    <ChartClickAction
-                      key={index}
-                      action={action}
-                      isLastItem={index === actions.length - 1}
-                      handleClickAction={this.handleClickAction}
-                    />
-                  ))}
+                  {SECTIONS[key].icon === "sum" && (
+                    <p className="mt0 text-medium text-small">{t`Summarize`}</p>
+                  )}
+                  {SECTIONS[key].icon === "breakout" && (
+                    <p className="my1 text-medium text-small">{t`Break out by a…`}</p>
+                  )}
+                  {SECTIONS[key].icon === "bolt" && (
+                    <p className="mt2 text-medium text-small">
+                      {t`Automatic explorations`}
+                    </p>
+                  )}
+                  {SECTIONS[key].icon === "funnel_outline" && (
+                    <p
+                      className={cx(
+                        "text-small",
+                        hasOnlyOneSection ? "mt0" : "mt2",
+                        hasOnlyOneSection ? "text-dark" : "text-medium",
+                      )}
+                    >
+                      {t`Filter by this value`}
+                    </p>
+                  )}
+
+                  <div
+                    className={cx(
+                      "flex",
+                      {
+                        "justify-end": SECTIONS[key].icon === "gear",
+                      },
+                      {
+                        "align-center justify-center":
+                          SECTIONS[key].icon === "gear",
+                      },
+                      { "flex-column my1": SECTIONS[key].icon === "summarize" },
+                    )}
+                  >
+                    {actions.map((action, index) => (
+                      <ChartClickAction
+                        key={index}
+                        action={action}
+                        isLastItem={index === actions.length - 1}
+                        handleClickAction={this.handleClickAction}
+                      />
+                    ))}
+                  </div>
                 </div>
-              </div>
-            ))}
-          </div>
-        )}
-      </Popover>
+              ))}
+            </div>
+          )
+        }
+        {...popoverAction?.popoverOptions}
```

Comment:
Q: Is this more like `popoverProps`? I'd expect `options` would be something that you pass as a whole object, but spreading everything in `popoverOptions` make it seems like this behave more like a React component props. What do you think?

Reply:
Agreed! Changed to `popoverProps`